pyproject.toml

```diff
@@ -13,6 +13,8 @@ pytest = "^7.0.1"
 selenium = "^4.1.2"
 markupsafe = "2.0.1"
 gunicorn = "^20.1.0"
+pymongo = {extras = ["srv"], version = "^4.2.0"}
```

Comment:
You shouldn't need the `srv` extra with CosmosDB (a previous version of the project exercise that used MongoDB Atlas did)